internal/pkg/deploy/cloudformation/stack/app.go

```diff
@@ -54,9 +54,9 @@ const (
 	appAdminRoleParamName         = "AdminRoleName"
 	appExecutionRoleParamName     = "ExecutionRoleName"
 	appDNSDelegationRoleParamName = "DNSDelegationRoleName"
-	appOutputKMSKey               = "KMSKeyARN"
-	appOutputS3Bucket             = "PipelineBucket"
-	appOutputECRRepoPrefix        = "ECRRepo"
+	appOutputKMSKey               = "KMSKeyARN"      // Linked to AppRegionalResources.KMSKeyARN.
+	appOutputS3Bucket             = "PipelineBucket" // Linked to AppRegionalResources.S3Bucket.
+	appOutputECRRepoPrefix        = "ECRRepo"        // Linked to AppRegionalResources.RepositoryURLs.
```

Comment:
Can we remove these comments? I'm a bit hesitant to add comments for how things are linked together as those comments can become stale quickly, however feel free to change them to _what_ these consts represent

For example,
```
	appOutputKMSKey               = "KMSKeyARN"      // Name of the CloudFormation Output that holds the KMS Key ARN to encrypt artifact buckets.
```